homeassistant/components/connector/cover.py

```diff
@@ -0,0 +1,295 @@
+"""Support for Motion Blinds using their WLAN API."""
+
+import logging
+
+from connectorlocal.connectorlocal import (
+    ONEWAYWIRELESSMODE,
+    TWOWAYWIRELESSMODE,
+    VENETIANTYPE,
+    WIFIMOTORTYPE,
+)
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    ATTR_TILT_POSITION,
+    CoverDeviceClass,
+    CoverEntity,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, KEY_COORDINATOR, KEY_GATEWAY, MANUFACTURER
+
+POSITION_DEVICE_MAP = {
+    1: CoverDeviceClass.SHADE,
+    2: CoverDeviceClass.SHUTTER,
+    3: CoverDeviceClass.SHADE,
+    4: CoverDeviceClass.SHADE,
+    6: CoverDeviceClass.SHUTTER,
+    7: CoverDeviceClass.GATE,
+    8: CoverDeviceClass.AWNING,
+    10: CoverDeviceClass.SHADE,
+    11: CoverDeviceClass.SHADE,
+    12: CoverDeviceClass.CURTAIN,
+    13: CoverDeviceClass.CURTAIN,
+    14: CoverDeviceClass.CURTAIN,
+    26: CoverDeviceClass.SHADE,
+    43: CoverDeviceClass.SHUTTER,
+}
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the Motion Blind from a config entry."""
+    entities = []
+    connector = hass.data[DOMAIN][config_entry.entry_id][KEY_GATEWAY]
+    coordinator = hass.data[DOMAIN][config_entry.entry_id][KEY_COORDINATOR]
+    device_list = await connector.device_list()
+    if device_list is None:
+        return
```

Comment:
If this happens, what do we do? Should we inform the user?

Reply:
Thank you for your suggestion, this step is not necessary in theory, because it has been judged in the config flow interface